.github/workflows/test.yml

```diff
@@ -9,12 +9,16 @@ jobs:
   tests-main:
     name: Tests
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node: ["lts/fermium", "lts/gallium", 18]
```

Comment:
@wemeetagain do we want to support NodeJS v14?

@dadepo Please add a comment like
```
lts/fermium = v14
lts/gallium = 16
-- | --



```

Reply:
I believe we had previous reports such as #4035 where not using Node v16 had issues. As of Nov 2021, active LTS was set at Node v16: https://github.com/ChainSafe/lodestar/pull/3440 and I don't believe there was any intent to keep supporting anything previous to v16?

Reply:
I think we drop support for 14 since its no longer supported upstream.

Also ideally we use numbers instead of release names for the node versions. fermium/gallium/etc is more confusing than it needs to be XD

Reply:
Yup. v14 is not supported. This is already an outdated diff, with version 14 removed in the commit [here](https://github.com/ChainSafe/lodestar/pull/4142/commits/c28e777b041e8942e94a137c90f76396d0de57ae)

@wemeetagain Indeed, version numbers are more straight-forward. I'll update existing usage of release names to version numbers.